crates/context/src/lib.rs

```diff
@@ -263,6 +242,55 @@ impl ContextManager {
         Ok(fs::read(&path)?)
     }
 
+    pub fn get_application_latest_version(
+        &self,
+        application_id: &calimero_primitives::application::ApplicationId,
+    ) -> eyre::Result<semver::Version> {
+        let Some((version, _)) = self.get_latest_application_info(application_id) else {
+            eyre::bail!("failed to get application with id: {}", application_id)
+        };
+
+        Ok(version)
+    }
+
+    async fn boot(&self) -> eyre::Result<()> {
+        let handle = self.store.handle();
+
+        let mut iter = handle.iter(&calimero_store::key::ContextMeta::new([0; 32].into()))?;
+
+        for key in iter.keys() {
+            self.subscribe(&key.context_id()).await?;
+        }
+
+        Ok(())
+    }
+
+    async fn subscribe(
+        &self,
+        context_id: &calimero_primitives::context::ContextId,
+    ) -> eyre::Result<()> {
+        self.network_client
+            .subscribe(calimero_network::types::IdentTopic::new(context_id))
+            .await?;
+
+        info!(%context_id, "Subscribed to context");
+
+        Ok(())
+    }
+
+    async fn unsubscribe(
+        &self,
+        context_id: &calimero_primitives::context::ContextId,
+    ) -> eyre::Result<()> {
+        self.network_client
+            .unsubscribe(calimero_network::types::IdentTopic::new(context_id))
+            .await?;
+
+        info!(%context_id, "Unsubscribed from context");
+
+        Ok(())
+    }
```

Comment:
these network methods are in-between application methods

Reply:
you can use a different `impl` block for maintenance methods at the top (including `start`)